src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -551,7 +551,7 @@ If the associated font is a [variable font](https://developer.mozilla.org/en-US/
 weight: "100 900"
 ```
 
-If left undefined, Astro will try to infer the value based on the first [`source`](#src).
+If the value is set to `undefined`, Astro will try to infer the value based on the first [`source`](#src).
```

Comment:
I think the idea of "left as default" is important, because no one will actively "set" a value of `undefined`.

There could be a better way to say this though! Perhaps:

```suggestion
When a value is not set, by default Astro will try to infer the value based on the first [`source`](#src).
```

Reply:
```suggestion
When the value is not set, by default Astro will try to infer the value based on the first [`source`](#src).
```
I would say it like this, I kept `the` since we refer to the current property (eg. weight)

Reply:
Thanks for good point!